examples/python-dataflow/requirements.txt

```diff
@@ -6,7 +6,7 @@ ultralytics
 gitpython
 ipython  # interactive notebook
 matplotlib>=3.2.2
-numpy>=1.18.5
+numpy>=1.18.5,<2.0.0
```

Comment:
This is caused by the numpy 2 release, right? I'm also seeing the same CI errors for #558 .

Reply:
Yes! I'm going to push a fix on this branch

Reply:
Thanks a lot!

Looks like it's related to `cv2`: https://github.com/opencv/opencv-python/issues/997